app/src/main/java/com/example/sabonit/ProductActivity.java

```diff
@@ -69,7 +71,7 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
         Bundle bundle = getIntent().getExtras();
         db = FirebaseFirestore.getInstance();
         // default value when initializing the seek bar
-        initialProgress = 0;
+        initialProgress = 1;
```

Comment:
it's smart, exaactly what we talked about

Reply:
OK nice, we can check if there is min value for seek bar